Modelica/Mechanics/Translational.mo

```diff
@@ -3677,8 +3677,9 @@ is not taken into account.
       annotation (Icon(coordinateSystem(preserveAspectRatio=false), graphics={
         Line(points={{-80,-70},{80,-70}}, color={0,0,0}),
         Line(points={{-80,0},{85.607,-1.19754}},
-                                          color={0,0,0}, origin={-3,-49}, rotation=15),
-        Ellipse(extent={{-130,-120},{-30,-20}}, lineColor={0,0,0}, startAngle=0, endAngle=15),
+          color={0,0,0}, origin={-3,-49}, rotation=15),
+        Ellipse(extent={{-130,-120},{-30,-20}}, lineColor={0,0,0}, startAngle=0, endAngle=15,
+          closure=EllipseClosure.None),
```

Comment:
@tobolar EllipseClosure is legal Modelica 3.4. Why was it removed here?

Reply:
Looking at the icon, the arc/ellipse is located between the two lines of the "road". It represents the inclination angle. As the two existing lines close the angle already, the additional closure of the ellipse would imply the lines being doubled. Just look at the following icon's cutoff (Dymola). 
![grafik](https://user-images.githubusercontent.com/9588978/71963371-c7fcb080-31fb-11ea-968b-eb36fb5c7032.png)

Reply:
Ah, sorry. I was wrong here.